apps/dashboard/app/javascript/packs/files/fileops.js

```diff
@@ -0,0 +1,377 @@
+import {Swal, alertError, doneLoading, loading} from './sweet_alert.js';

```

Comment:
Does any other file use `sweet_alert.js`? That rephrased - maybe sweet_alert can just be here.

Maybe a pattern like this where we mixin, then use that variable. That seems to be what the docs indicate they want you to do anyhow.
```javascript
const FileOpAlert = Swal.mixin( ... );

# down on line 38 we then use this const instead of Swal.
  $('#new-dir-btn').on("click", () => {
    FileOpAlert.fire({
```

Reply:
Actually I see it being used elsewhere. Then maybe we can flip it - create `FileOpAlert` in a seperate file and export it there.

I'll have to dig into this a bit more and see how those other exported functions in `sweet_alert` relate to `Swal`.